docker/apply/views/help/terms-and-conditions.html

```diff
@@ -1,154 +1,216 @@
-{% extends 'layout.html' %}
```

Comment:
I think your formatter's gone to town on this file :-) Can you undo the formatting changes here to make it more readable?

Reply:
Turns out this formatting was being done automatically with pre commit hooks each time I comitted. Looks like we might have to create a ticket or PR just to push with the formatting change in case more changes need to be made to that this issue doesn't happen again.

Reply:
No problem.  At least it should be consistent going forwards.